examples/object-methods/object_methods.bal

```diff
@@ -13,7 +13,7 @@ type Person object {
         self.lastName = lastName;
     }
 
-    // A method defined within the object.
+    // A method.
```

Comment:
Can we make this comment more descriptive? If not let's remove this comment as it's quite obvious.

Reply:
This is a BBE about object methods. Made it more descriptive.